fenix/app/nimbus.fml.yaml

```diff
@@ -105,10 +105,10 @@ import:
                 surface: homescreen
                 action: "MAKE_DEFAULT_BROWSER"
                 trigger: [ "I_AM_NOT_DEFAULT_BROWSER","USER_ESTABLISHED_INSTALL" ]
-                style: "PERSISTENT"
+                style: PERSISTENT
```

Comment:
@jhugman presumably this shouldn't have had the surrounding quotes, or would it still work the same regardless?

Reply:
This will work, but we should proabably omit as many optional quotes as possible.